x/treasury/handler.go

```diff
@@ -50,12 +56,20 @@ func handleMsgOracleTransfer(ctx sdk.Context, k keeper.Keeper, msg types.MsgOrac
 		return err.Result()
 	}
 
-	ctx.EventManager().EmitEvent(
+	ctx.EventManager().EmitEvents(sdk.Events{
+		sdk.NewEvent(
+			types.EventTypeMsgSend,
```

Comment:
This should be a new event type `EventTypeOracleTransfer`

Reply:
ok. Updated the code 

ctx.EventManager().EmitEvents(sdk.Events{
		sdk.NewEvent(
			types.EventTypeOracleTransfer,
			sdk.NewAttribute(types.AttributeKeyOracleDid, msg.OracleDid),
			sdk.NewAttribute(types.AttributeKeyTxHashFromDid, msg.FromDid),
			sdk.NewAttribute(types.AttributeKeyToDidOrAddr, msg.ToDidOrAddr),
			sdk.NewAttribute(types.AttributeKeyAmount, msg.Amount.String()),
			sdk.NewAttribute(types.AttributeKeyProof, msg.Proof),
		),